core/extractors/extractors_latex.py

```
@@ -0,0 +1,532 @@
+        """Convert internal dict format to ExtractionResult."""
+        structures = data.get('structures', {})
+        return ExtractionResult(
+            text=data.get('full_text', ''),
+            metadata=data.get('metadata', {}),
+            equations=structures.get('equations', []),
+            tables=structures.get('tables', []),
+            references=structures.get('citations', []),
+            processing_time=data.get('metadata', {}).get('processing_time', 0.0)
+        )
+
+    def extract(self, latex_path: Union[str, Path], **kwargs) -> ExtractionResult:
```

Ruff (0.14.13)

Unused method argument: `kwargs`

(ARG002)